packages/perseus/src/components/highlighting/ui/highlight-tooltip.jsx

```diff
@@ -94,31 +92,36 @@ class HighlightTooltip extends React.PureComponent<HighlightTooltipProps> {
             return null;
         }
 
-        const content = (
-            <div className={css(styles.tooltipLabel)}>{this.props.label}</div>
+        // using div instead of TooltipContent because
+        // TooltipContent wouldn't let me overwrite
+        // user-select and onClick
+        const content: $FlowFixMe = (
```

Comment:
Yeah, Flow related to Tooltip throws an opaque error: "Cannot create `Tooltip` element because indexed  object type [1] is incompatible with exact  `Props` [2] in property `props` of property `content`.Flow(incompatible-type)"

I tried a few things, but ended up moving on since we're closing in on a deadline.